spec/lib/ddq-backend-mysql.spec.js

```diff
@@ -168,33 +168,33 @@ describe("lib/ddq-backend-mysql", () => {
             instance.connection.query.andCallFake((query, option, callback) => {
                 callback({
                     code: "ER_DUP_ENTRY"
-                });
+                }, {});
             });
-            instance.sendMessage("Example Message", () => {}, "Some Topic");
+            instance.sendMessage("Example Message", "Some Topic", () => {});
             expect(instance.emit.callCount).toBe(1);
         });
         describe(".trySendMessage()", () => {
             it("emits an error", () => {
                 instance.connection.query.andCallFake((query, option, callback) => {
                     callback({
                         Error: "Some Error"
-                    });
+                    }, {});
                 });
-                instance.sendMessage("Example Message", () => {});
+                instance.sendMessage("Example Message", null, () => {});
```

Comment:
This should work with and without the `null`.

    instance.sendMessage("Example Message", null, () => {});
    instance.sendMessage("Example Message", () => {});